index.html

```diff
@@ -27,11 +29,71 @@
 <!-- Estimate Your Project Section [Esraa Elshibly]-->
 <section class="estimate-project section--py section"></section>
 
+
+
+
 <!-- My Recent Works Section [Alaa]-->
-<section class="recent-works section--py section"></section>
+<section class="recent-works">
+
+      <h1>My Recent Works</h1>
+      <ul>
+        <li>ALL WORKS</li>
+        <li>UI/UX</li>
+        <li>BRANDING</li>
+        <li>LOGO</li>
+        <li>DEVELOPMENT</li>
+      </ul>
+
+   <div class="img">
+
+     <img src="../portfolio/assets/imgs/recentWorks/rw1.png">
+     <img src="../portfolio/assets/imgs/recentWorks/rw2.png">
+     <img src="../portfolio/assets/imgs/recentWorks/rw3.png">
+     <img src="../portfolio/assets/imgs/recentWorks/rw4.png">
+     <img src="../portfolio/assets/imgs/recentWorks/rw5.png">
+     <img src="../portfolio/assets/imgs/recentWorks/rw6.png">
+
+   </div>
+            <button>MORE PROJECTS</button>
```

Comment:
I think when u click More project it will go to Projects page so i think using anchor instead of button